src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -127,6 +130,7 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
               relative: getRelativeDate(dateAdded, 'ago', true),
             }),
         };
+      case GroupInboxReason.NEW:
```

Comment:
was this just added to be more explicit?

Reply:
i'm making some references to GroupInboxReason.INBOX so i added constants for all of them.

Reply:
oh, yeah. I was looking for NEW and couldn't find it.